Traceback (most recent call last):
  File "badly.py", line 7, in <module>
    inverse(0)
  File "badly.py", line 4, in inverse
    return 1 / x
ZeroDivisionError: division by zero
